src/Uno.UI/Controls/Window.iOS.cs

```diff
@@ -80,9 +87,9 @@ public override void PressesBegan(NSSet<UIPress> presses, UIPressesEvent evt)
 		{
 			var handled = false;
 			var focusInputHandler = Uno.UI.Xaml.Core.CoreServices.Instance.MainRootVisual?.AssociatedVisualTree?.UnoFocusInputHandler;
-			if (Uno.WinRTFeatureConfiguration.Focus.EnableExperimentalKeyboardFocus && focusInputHandler != null)
```

Comment:
Is there a reason to invert the two checks? For me we are going to check the feature flag for each touch while we could do it only once.

Reply:
Updated!